### PR TITLE
Docs/subscription

### PR DESCRIPTION
**Pull request by Urigo**

### CODE REVIEW

File: source/subscriptions.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+const withData = graphql(COMMENT_QUERY, {
+    name: 'comments',
+    options: ({ params }) => ({
+        variables: {
+            repoName: `${params.org}/${params.repoName}`
+        },
+    }),
+    props: props => {
+        return {
+            ...props,
```
Comment (stubailo) [inline source/subscriptions.md]: We already merge the props so there's no need to do this.

**Comment by Urigo** [comment]
@helfer I've re-ordered the article like you suggested.
I definitely better now than was before

**Comment by helfer** [comment]
@Urigo Looks good! I made a bunch more edits. If you're okay with them, you can merge the PR! 🙂

**Comment by calebmer** [comment]
There are also some `TODO`s around subscriptions in the API docs if you want to fill them in.

- https://github.com/apollographql/react-docs/blob/3ccc90c06ca10e245c20e8ddec18730675a0d1c5/source/api.md#propsdatasubscribetomoreoptions
- https://github.com/apollographql/react-docs/blob/3ccc90c06ca10e245c20e8ddec18730675a0d1c5/source/api.md#subscriptions

**Comment by Urigo** [comment]
@calebmer ok, I'm merging and will open in a new PR

**Pull request merged**